docs/platforms.md

```diff
@@ -78,35 +73,19 @@ module.exports = {
 On Android and iOS, this function returns:
 
 ```ts
-type ProjectConfigIOST = {
+interface IOSProjectConfig {
   sourceDir: string;
-  folder: string;
-  pbxprojPath: string;
-  podfile: null;
-  podspecPath: null;
-  projectPath: string;
-  projectName: string;
-  libraryFolder: string;
-  sharedLibraries: Array<any>;
-  plist: Array<any>;
-};
+  scriptPhases: Array<any>;
 
-type ProjectConfigAndroidT = {
+  podfile?: string;
```

Comment:
Looks like podfile is always there in iOS project config, but may be undefined. This type would make it more accurate:
```suggestion
  podfile: string | undefined;
```